osu.Game/Screens/Select/BeatmapPlaylistItem.cs

```diff
@@ -0,0 +1,311 @@
+using osu.Framework.Input.Events;
+using osu.Framework.Localisation;
+using osu.Game.Beatmaps.Drawables;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online.Multiplayer;
+using osuTK;
+using osuTK.Graphics;
+using osuTK.Input;
+
+namespace osu.Game.Screens.Select
+{
+    public class BeatmapPlaylistItem : Container
+    {
+        public readonly Bindable<PlaylistItem> PlaylistItem = new Bindable<PlaylistItem>();
+
+        public event Action<BeatmapPlaylistItem> RequestRemoval;
+
+        private const int fade_duration = 60;
+        private readonly DragHandle dragHandle;
+        private readonly RemoveButton removeButton;
+        private bool isHovered;
+        private bool isDragged;
+
+        public BeatmapPlaylistItem(PlaylistItem item)
+        {
+            UpdateableBeatmapBackgroundSprite cover;
+            Height = 50;
+            RelativeSizeAxes = Axes.X;
+            Children = new Drawable[]
+            {
+                new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Padding = new MarginPadding
+                    {
+                        Left = 25,
+                    },
+                    Child = new Container
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Masking = true,
+                        CornerRadius = 5,
+                        EdgeEffect = new EdgeEffectParameters
+                        {
+                            Type = EdgeEffectType.Shadow,
+                            Colour = Color4.Black.Opacity(40),
+                            Radius = 5,
+                        },
+                        Children = new Drawable[]
+                        {
+                            new FillFlowContainer
+                            {
+                                RelativeSizeAxes = Axes.Both,
+                                Children = new Drawable[]
+                                {
+                                    new Box
+                                    {
+                                        Colour = Color4.Black,
+                                        RelativeSizeAxes = Axes.Both,
+                                        Width = 0.25f,
+                                    },
+                                    new Container
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Width = 0.75f,
+                                        Children = new Drawable[]
+                                        {
+                                            cover = new UpdateableBeatmapBackgroundSprite
+                                            {
+                                                RelativeSizeAxes = Axes.Both,
+                                                FillMode = FillMode.Stretch
```

Comment:
this fillmode is likely not doing anything (removing it results in no behavioural change).